client/components/credit-card/stored-card.tsx

```diff
@@ -123,26 +146,35 @@ const StoredCard: FunctionComponent< Props > = ( {
 	lastDigits,
 	name,
 	selected,
+	email,
+	paymentPartner,
 } ) => {
 	const translate = useTranslate();
 	const moment = useLocalizedMoment();
 
 	// The use of `MM/YY` should not be localized as it is an ISO standard across credit card forms: https://en.wikipedia.org/wiki/ISO/IEC_7813
-	const expirationDate = expiry ? moment( expiry, moment.ISO_8601, true ).format( 'MM/YY' ) : null;
+	const expirationDate = expiry ? moment( expiry, moment.ISO_8601, true ) : null;
+	const displayExpirationDate =
+		expirationDate && expirationDate.isValid() ? expirationDate.format( 'MM/YY' ) : null;
```

Comment:
This is totally fine but just so you know we can use optional chaining in calypso, so this also could be:

```js
const displayExpirationDate = expirationDate?.isValid() ? expirationDate.format( 'MM/YY' ) : null;
```